package.json

```diff
@@ -62,7 +62,7 @@
     "m3u8-parser": "4.7.0",
     "mpd-parser": "0.19.0",
     "mux.js": "5.13.0",
-    "video.js": "^6 || ^7"
+    "video.js": "^7.15.4"
```

Comment:
we should revert this change unless we really do depend on something that requires 7.15.4 at this point.

Reply:
Yeah I reverted it in the merge that I had to do. I updated video.js in the `package-lock.json` though.

Reply:
should've waited for the rebase then haha